src/routes/Routes.js

```diff
@@ -1,20 +1,22 @@
 import React from 'react';
 import { Route, Switch, Redirect } from 'react-router-dom';
-import BrowserProvider from '../util/URL/BrowserProvider';
+import BrowserProvider from '../util/browser/BrowserProvider';
 
+import RouteValidator from '../auth/routes/Validator';
 import TodoList from '../pages/Todo/List/TodoList';
+import TodoEdit from '../pages/Todo/Edit/TodoEdit';
 import NotFound from '../pages/Fallback/NotFound';
 import Buggy from '../pages/Fallback/Buggy';
-import RouteValidator from '../auth/Routes/Validator';
 
 const Routes = () => {
```

Comment:
Since this component does not have state or logic and it's just a simple return it can be simplified to:

```js
const Routes = () => (
  <Switch>
  // routes
  </Switch>
)
```

Reply:
I think we would have to refactor all the code to keep this pattern. IMO it does not aggregate that much changing it. I will leave as it is for now. Tks for the comment though.